packages/dht/src/dht/routing/FindRpcRemote.ts

```diff
@@ -0,0 +1,38 @@
+import { RouteMessageWrapper } from '../../proto/packages/dht/protos/DhtRpc'
+import { v4 } from 'uuid'
+import { keyFromPeerDescriptor } from '../../helpers/peerIdFromPeerDescriptor'
+import { Remote } from '../contact/Remote'
+import { Logger } from '@streamr/utils'
+import { IFindRpcClient } from '../../proto/packages/dht/protos/DhtRpc.client'
+
+const logger = new Logger(module)
+
+export class FindRpcRemote extends Remote<IFindRpcClient> {
+
+    async findRecursively(params: RouteMessageWrapper): Promise<boolean> {
```

Comment:
findRecursively can simply be called find

Reply:
Renamed